### ISSUE TITLE
Question: is this for Electron only?

### ISSUE THREAD

**Issue by andrispraulitis** [tojocky/node-printer#286]
How to install directly from git and use this simple react web app?

**Comment by DanMMX** [comment]
No, this is not for Electron only, you can use it on NodeJS as well. React is client side, you won't be able to use this module unless you communicate with some NodeJS server.

**Comment by andrispraulitis** [comment]
Thanks @DanMMX , I'm new to this printer stuff. Basically I have a React webapp that has raw epl/zpl format label I need to send to the printer. Wondering if it is at all possible directly from client(browser)...

**Comment by DanMMX** [comment]
Nope. Apps from the browser can't access hardware, unless you make it an Electron app.
